patroni.py

```diff
@@ -88,6 +91,11 @@ def run(self):
             try:
                 if self.ha.state_handler.is_leader():
                     self.ha.cluster and self.ha.state_handler.create_replication_slots(self.ha.cluster)
+
+                    # SkyDNS2 support: publish leader
```

Comment:
We'd like to keep the code as generic as possible. This could be implemented by the ACTION_ON_ROLE_CHANGE callback (to call a script for example). This does however require some more documentation